frontend/src/components/common/header/header-container.tsx

```diff
@@ -0,0 +1,110 @@
+import { FC } from 'common/types/types';
+import { Link } from 'react-router-dom';
+import { FormEvent, MouseEvent, RefObject } from 'react';
+
+import { AppRoute, IconName } from 'common/enums/enums';
+import { MenuOptions } from 'common/enums/components';
+
+import styles from './header.module.scss';
+import { Icon } from '../icon';
```

Comment:
Why do you have empty line above styles import? If you want to separate scss import it has to be last import